content/reference/endpoints/bookings_taxes.md

```diff
@@ -0,0 +1,43 @@
+# Bookings Taxes
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | Bookings Tax's id.
+-----------------|---------|------------|------------
+amount           | [Decimal](/reference/enums#formats)  | Read       | Bookings Tax's amount.
+name             | [Object](/reference/enums#formats)   | Read       | Bookings Tax's name, list of locales described in [enums section](/reference/enums#locales).
+percentage       | [Decimal](/reference/enums#formats)  | Read       | Bookings Tax's percentage.
+tax_included_in_price | Boolean | Read       | Tax amount already included in bookings price if set to true.
```

Comment:
It has `already` in the description, suggesting it was not added with the creation of the Tax. Or I got this wrong?
Taxable`s price sounds completely voodoo to me :D